tmpl/README.md.j2

```diff
@@ -12,25 +12,54 @@ and have no additional dependencies. Moreover we distinguish pinned installation
 is based on first level dependencies listed in `requirements/requirements.yml` and `requirements/dev-requirements.yml` respectively. If you start developing, you might want to do a free
 development installation and export the environments right away.
 ```bash
-setup_env.sh -d -e -m -n <package_env_name>
+setup_env.sh -u -d -e -n <package_dev_env_name>
```

Comment:
Maybe mention the following situation just to save the user some time. I ran into this issue several times with completely independent installations:
```
An error has occurred: InvalidManifestError: 
==> File /users/jappel/.cache/pre-commit/repozhhauwxe/.pre-commit-hooks.yaml
==> At Hook(id='black-jupyter')
==> At key: types_or
==> At index 2
=====> Type tag 'jupyter' is not recognized.  Try upgrading identify and pre-commit?
Check the log at /users/jappel/.cache/pre-commit/pre-commit.log
```
This occured after running `setup_env.sh` when I wanted to run `pre-commit`. Force-re-installing `pre-commit` fixed it for me every time:
```bash
pip install --force-reinstall -U pre-commit
```
If this is not a bug that can be fixed easily, I would add a small note here, mentioning that if one runs into this issue, this is the way to fix it.